### 单独使用 Panic 函数

你也可以单独使用`panic()`函数，而无需和**recover**配合，该小节将使用`justPanic.go`的 Go 代码显示其结果，该代码分为两部分。

第一部分代码：

```Go
package main
import (
  "fmt"
  "os"
)
```

使用`panic()`不要求引入任何的 Go package。

第二部分代码：

```Go
func main() {
  if len(os.Args) == 1 {
    panic("Not enough arguments!")
  }
  fmt.Println("Thanks for the argument(s)!")
}
```

如果你的 Go 程序没有接收一个命令行参数，它将调用`panic()`函数。`panic()`函数需要一个参数，这是你要在屏幕上打印的错误消息。

在 macOS Mojave 机器上执行`justPanic.go`将输出：

```shell
$ go run justPanic.go
panic: Not enough arguments!
goroutine 1 [running]:
main.main()
  /Users/mtsouk/ch2/code/justPanic.go:10 +0x91
exit status 2
```

因此，单独使用`panic()`函数将终止 Go 程序，而不会给你处理异常机会。因此，组合使用`panic()`和`recovery()`比仅使用`panic()`更实用且显得专业。

> Tip: `panic()`函数的输出类似于日志包中`Panic()`函数的输出。但是，`panic()`函数不会将任何内容发送到 UNIX 计算机的日志记录服务。
